src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './TodoForm.css';
+
+const TodoForm = props => {
+  let input = React.createRef();
+
+  return (
+    <div className="todo-form">
+      <div className='field'>
+        <p>CREATE TODO:</p>
+        <input ref={input} />
```

Comment:
this is an interesting way to setup your form input, but i would suggest that a changehandler function that stores your input data on state may be a more flexible option to explore, especially when you start building out forms with more input fields.